lib/git-log.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
 
     activate: function() {
-        atom.workspaceView.command("git-log:show", function(event) {
+        atom.views.getView(atom.workspace).command("git-log:show", function(event) {
```

Comment:
I think this is the way to add commands.
```js
atom.commands.add("atom-workspace", "git-log:show", function(event) {
  ...
});
```
also `addOpener`should be used instead of `registerOpener`